tests/functional/u/use/use_maxsplit_arg.txt

```diff
@@ -1,21 +1,21 @@
-use-maxsplit-arg:5:12::Use '1,2,3'.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:6:11::"Use '1,2,3'[::-1].split(',', maxsplit=1)[0] instead"
-use-maxsplit-arg:9:12::Use SEQ.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:10:11::Use SEQ.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:11:12::Use SEQ.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:12:11::Use SEQ.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:45:12::Use Foo.class_str.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:46:11::Use Foo.class_str.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:47:12::Use Foo.class_str.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:48:11::Use Foo.class_str.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:56:12::Use bar.get_string().split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:57:11::Use bar.get_string().rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:66:10::Use s.split(' ', maxsplit=1)[0] instead
-use-maxsplit-arg:67:10::Use s.rsplit(' ', maxsplit=1)[-1] instead
-use-maxsplit-arg:76:6::Use Bar.split.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:77:6::Use Bar.split.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:78:6::Use Bar.split.split(',', maxsplit=1)[0] instead
-use-maxsplit-arg:79:6::Use Bar.split.rsplit(',', maxsplit=1)[-1] instead
-use-maxsplit-arg:82:4::Use '1,2,3'.split('\n', maxsplit=1)[0] instead
-use-maxsplit-arg:83:4::Use '1,2,3'.rsplit('split', maxsplit=1)[-1] instead
-use-maxsplit-arg:84:4::Use '1,2,3'.split('rsplit', maxsplit=1)[0] instead
+use-maxsplit-arg:5:12::Use '1,2,3'.split(',', maxsplit=1)[0] instead:HIGH
```

Comment:
Just my opinion, but I'm not a big fan of adding `:HIGH` and creating larger diffs than necessary. That makes it quite difficult to review.